go-controller/pkg/config/config.go

```diff
@@ -31,8 +31,10 @@ const DefaultAPIServer = "http://localhost:8443"
 
 // IP address range from which subnet is allocated for per-node join switch
 const (
-	V4JoinSubnet = "100.64.0.0/16"
-	V6JoinSubnet = "fd98::/64"
+	V4JoinSubnet          = "100.64.0.0/16"
+	V6JoinSubnet          = "fd98::/64"
+	V4HybridOverlaySubnet = "10.132.0.0/14/23"
+	V6HybridOverlaySubnet = "fd99::/64/128"
```

Comment:
The join subnet has v4/v6 values because it's not overridable. We don't have separate IPv4 and IPv6 defaults for user-specifiable subnets. You have to just specify a non-default value if you want to use IPv6.
(Which means this whole commit can go away.)

Reply:
I should have added more info to the commit description. I broke this into it's own commit at the last minute. Sorry.
The issue is the `checkIPFamilies()` in `go-controller/pkg/config/utils.go` is using:
```
	if len(cs.v6) == 0 {
		// Single-stack IPv4
		return true, false, nil
	} else if len(cs.v4) == 0 {
		// Single-stack IPv6
		return false, true, nil
	}
:
	return false, false, fmt.Errorf("illegal network configuration: %s", netConfig)
```
Since hybrid-overlay defaults to IPv4, and `checkIPFamilies()` doesn't check to see if hybrid-overlay is enabled, it fails in IPv6 only with:
```
F0610 17:46:37.052861    2852 ovnkube.go:129] illegal network configuration: IPv6 cluster subnet, IPv6 service subnet, IPv4 hybrid overlay subnet
```
I liked the simplicity of `checkIPFamilies()` so I changed how hybrid overlay was defaulted, default to empty then if hybrid overlay enabled and IP address not set, then set to IPv4 or IPv6 default IP Address.

Reply:
ah... no, it's better for `checkIPFamilies` to check if hybrid overlay is enabled

Reply:
Moved the fix over to `checkIPFamilies` and added some debug, and haven't been able to detect the original problem. After a day of trying different things, I'll just remove this commit and save off the diffs if it is seen in the future.